expyfun/tests/test_experiment_controller.py

```diff
@@ -399,3 +399,32 @@ def test_background_color():
         black_mask = (ss == [0] * 3)
         assert_true(black_mask.any())
         assert_true(np.logical_or(gray_mask, black_mask).all())
+
+
+@_hide_window
+def test_tdt_delay():
+    """test the tdt_delay parameter"""
+    with ExperimentController(*std_args, tdt_delay=0, audio_controller='tdt',
+                              **std_kwargs) as ec:
+        ec.wait_secs(0)  # so pyflakes doesn't whine about not using ec
+    with ExperimentController(*std_args, tdt_delay=1, audio_controller='tdt',
+                              **std_kwargs) as ec:
+        ec.wait_secs(0)
+    with ExperimentController(*std_args, tdt_delay=None,
+                              audio_controller='tdt', **std_kwargs) as ec:
+        ec.wait_secs(0)
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter('always')
+        with ExperimentController(*std_args, tdt_delay=3,
+                                  audio_controller='pyglet', 
```

Comment:
trailing whitespace here is making travis flake out.  (my bad, but probably easier if @rkmaddox fixes)